website/components/card-list/index.tsx

```diff
@@ -0,0 +1,53 @@
+import s from './style.module.css'
+
+interface Card {
+  image: {
+    src: string
+    alt: string
+  }
+  description: string
+  cta: {
+    url: string
+    text: string
+  }
+  type: string
```

Comment:
_type_ slightly assumes the content here.  I'm actually not 100% sure what will go in here, but I am suspicious that it might be something that `type` will not read well with.

Maybe something generic to just describe the visual pattern, maybe like `eyebrow`?